scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -0,0 +1,304 @@
+    input:
+        entity_id=wf_entity_id_tumor,
+        coverage_file=TumorCalculateTargetCoverage.gatk_cnv_coverage_file,
+        padded_target_bed=PadTargets.padded_target_bed,
+        pon=PoN,
+        jar_file=jar_file,
+        mem=2,
+	jni_lib=jni_lib
+  }
+
+  call PerformSegmentation as TumorPerformSeg{
+    input:
+        entity_id=wf_entity_id_tumor,
+        jar_file=jar_file,
+        tn_file=TumorNormalizeSomaticReadCounts.tn_file,
+        mem=2
+  }
+
+  call Caller as TumorCaller {
+    input:
+        entity_id=wf_entity_id_tumor,
+        jar_file=jar_file,
+        tn_file=TumorNormalizeSomaticReadCounts.tn_file,
+        seg_file=TumorPerformSeg.seg_file,
+        mem=2
+  }
+
+  call CalculateTargetCoverage as NormalCalculateTargetCoverage {
+    input:
+        entity_id=wf_entity_id_normal,
+        padded_target_bed=PadTargets.padded_target_bed,
+        input_bam=tumor_bam,
+        ref_fasta=ref_fasta,
+        ref_fasta_fai=ref_fasta_fai,
+        ref_fasta_dict=ref_fasta_dict,
+        jar_file=jar_file,
+        disable_reference_validation=is_disable_reference_validation,
+        mem=2
+
+  }
+
+  call NormalizeSomaticReadCounts as NormalNormalizeSomaticReadCounts {
+    input:
+        entity_id=wf_entity_id_normal,
+        coverage_file=NormalCalculateTargetCoverage.gatk_cnv_coverage_file,
+        padded_target_bed=PadTargets.padded_target_bed,
+        pon=PoN,
+        jar_file=jar_file,
+        mem=2,
+	jni_lib=jni_lib
+  }
+
+  call PerformSegmentation as NormalPerformSeg {
+    input:
+        entity_id=wf_entity_id_normal,
+        jar_file=jar_file,
+        tn_file=NormalNormalizeSomaticReadCounts.tn_file,
+        mem=2
+  }
+
+  call Caller as NormalCaller {
+    input:
+        entity_id=wf_entity_id_normal,
+        jar_file=jar_file,
+        tn_file=NormalNormalizeSomaticReadCounts.tn_file,
+        seg_file=NormalPerformSeg.seg_file,
+        mem=2
+  }
+
+  call HetPulldown {
+    input:
+        entity_id_tumor=wf_entity_id_tumor,
+        entity_id_normal=wf_entity_id_normal,
+        jar_file=jar_file,
+        mem = 4,
+        ref_fasta=ref_fasta,
+        ref_fasta_fai=ref_fasta_fai,
+        ref_fasta_dict=ref_fasta_dict,
+        tumor_bam=tumor_bam,
+        tumor_bam_idx=tumor_bam_idx,
+        normal_bam=normal_bam,
+        normal_bam_idx=normal_bam_idx,
+        common_snp_list=common_snp_list
+  }
+
+  call AllelicCNV {
+    input:
+        entity_id=wf_entity_id_tumor,
+        jar_file=jar_file,
+        mem=4,
+        tumor_hets=HetPulldown.tumor_hets,
+        called_file=TumorCaller.called_file,
+        tn_file=TumorNormalizeSomaticReadCounts.tn_file
+  }
+}
+
+# Pad the target file.  This was found to help sensitivity and specificity.  This step should only be altered
+#  by advanced users.  Note that by changing this, you need to have a PoN that also reflects the change.
+task PadTargets {
+    File target_bed
+    Int pd = 250
```

Comment:
You can name the variable `padding` without running afoul of the verbosity police.